solidity/contracts/examples/Consumer.sol

```diff
@@ -25,25 +24,19 @@ contract Consumer is Chainlinked, Ownable {
     string[] memory path = new string[](1);
     path[0] = _currency;
     run.addStringArray("path", path);
-    requestId = chainlinkRequest(run, LINK(1));
+    chainlinkRequest(run, LINK(1));
```

Comment:
In light of the new events, perhaps this should be renamed to `requestChainlink`, so as not to coincide with `emit ChainlinkRequest`.